exosphere-shared/src/docker-helper.ls

```diff
@@ -84,5 +85,10 @@ class DockerHelper
       child_process.exec-sync 'docker rm -f $(docker ps -aq)'
 
 
+  # Prints the last n number of lines from a given text
+  @_print-last-lines = (text, number-of-lines) ->
+    split-text = text.split /\r?\n/
+    sliced-text = split-text.slice Math.max(split-text.length - number-of-lines - 1, 0)
+    sliced-text.join (if process.platform == 'win32' then '\r\n' else '\n')
```

Comment:
I think this can be
```livescript
@_print-last-lines = (text, number-of-lines) ->
  text
    .split '\n'
    .[-number-of-lines to]
    .join '\n'